tests/pass/simple.py

```diff
@@ -25,9 +25,26 @@ def ndarray_func(x):
 np.dtype(np.dtype(float))
 np.dtype(('U', 10))
 np.dtype((np.int32, (2, 2)))
-np.dtype([('R', 'u1'), ('G', 'u1'), ('B', 'u1')])
-np.dtype([('R', 'u1', 1)])
-np.dtype([('R', 'u1', (2, 2))])
+# Define the arguments on the previous line to prevent bidirectional
+# type inference in mypy from broadening the types.
+two_tuples_dtype = [('R', 'u1'), ('G', 'u1'), ('B', 'u1')]
+np.dtype(two_tuples_dtype)
+
+three_tuples_dtype = [('R', 'u1', 1)]
```

Comment:
Note that if you moved the dtype to the previous line to avoid broadening, this test would fail on master.